### ISSUE TITLE
Large Scale Fine-Grained Categorization and Domain-Specific Transfer Learning

### ISSUE THREAD

**Issue by sweaterr** [sweaterr/readinglist#337]
https://vision.cornell.edu/se3/wp-content/uploads/2018/03/FGVC_CVPR_2018.pdf

**Comment by sweaterr** [comment]
소개
* 데이터가 롱테일이다
  * 이를 다루기 위해 간단하고 효과적인 방법을 소개한다
* 전송 학습
  * 백본이 학습된 데이터가 타겟에 가까울수록 잘된다

관련 연구

**Comment by sweaterr** [comment]
## 3.2. Long-Tailed Distribution

The statistics of real world images is long-tailed: a few categories are highly representative and have most of the images, whereas most categories are observed rarely with only a few images [71, 56].
실세계 이미지의 통계는 긴 꼬리가있다. 몇 가지 카테고리는 매우 대표적이고 대부분의 이미지를 가지고 있지만, 대부분의 카테고리는 드물게 단지 몇 개의 이미지만으로 관찰된다 [71, 56].

This is in stark contrast to the even image distribution in popular benchmark datasets such as ImageNet [12], COCO [34] and CUB200 [58].
이는 ImageNet [12], COCO [34] 및 CUB200 [58]과 같은 널리 사용되는 벤치 마크 데이터 세트의 짝수 이미지 배포와는 현저한 대조를 이룹니다.

With highly imbalanced numbers of images across categories in iNaturalist dataset [55], we observe poor performance on underrepresented tail categories.
iNaturalist 데이터 세트 [55]의 범주간에 이미지의 불균형이 심하게 나타나기 때문에 미미한 꼬리 범주에 대한 성능이 좋지 않습니다.

We argue that this is mainly caused by two reasons: 1) The lack of training data.
우리는 이것이 주로 두 가지 이유로 인해 발생한다고 주장한다. 1) 훈련 자료가 부족하다.

Around 1,500 fine-grained categories in iNat training set have fewer than 30 images.
iNat 교육 세트의 약 1,500 개의 세부 카테고리에는 30 개 미만의 이미지가 있습니다.

2) The extreme class imbalance encountered during training: the ratio between the number of images in the largest class and the smallest one is about 435.
2) 훈련 중 발생하는 극한의 불균형 : 가장 큰 클래스와 가장 작은 클래스의 이미지 수의 비율은 약 435입니다.

Without any re-sampling of the training images or re-weighting of the loss, categories with more images in the head will dominate those in the tail.
훈련 이미지를 다시 샘플링하거나 손실을 다시 가중시키지 않으면 머리에 더 많은 이미지가있는 범주가 꼬리 부분을 지배하게됩니다.

Since there is very little we can do for the first issue of lack of training data, we propose a simple and effective way to address the second issue of the class imbalance.
훈련 데이터가 부족하다는 첫 번째 문제에 대해 우리가 할 수있는 일이 거의 없으므로 우리는 두 번째 문제인 클래스 불균형을 해결할 수있는 간단하고 효과적인 방법을 제안합니다.

The proposed training scheme has two stages.
제안 된 훈련 계획에는 두 단계가 있습니다.

In the first stage, we train the network as usual on the original imbalanced dataset.
첫 번째 단계에서는 원래의 불균형 데이터 세트에서 평소대로 네트워크를 교육합니다.

With large number of training data from all categories, the network learns good feature representations.
모든 범주의 많은 교육 데이터로 네트워크는 우수한 기능 표현을 학습합니다.

Then, in the second stage, we fine-tune the network on a subset containing more balanced data with a small learning rate.
그런 다음 두 번째 단계에서는보다 균형 잡힌 데이터가 포함 된 하위 집합에서 네트워크를 미세하게 학습합니다.

The idea is to slowly transfer the learned feature and let the network re-balance among all categories.
아이디어는 학습 된 기능을 천천히 전송하고 네트워크가 모든 카테고리간에 균형을 이루게하는 것입니다.

Fig  2 shows the distribution of image frequency in iNat training set that we trained on in the first stage and the subset we used in the second stage, respectively.
그림 2는 iNat 트레이닝 세트에서 이미지 주파수의 분포를 보여 주며 첫 번째 단계에서 트레이닝했으며 두 번째 단계에서 사용한 서브 세트를 각각 보여줍니다.

Experiments in Sec 5.2 verify that the proposed strategy yields improved overall performance, especially for underrepresented tail categories.
초 5.2의 실험은 제안 된 전략이 전반적인 성능을 향상 시킨다는 것을 증명한다.
